packages/worker/src/utilities/appService.js

```diff
@@ -0,0 +1,26 @@
+const fetch = require("node-fetch")
+const { Headers } = require("@budibase/auth/constants")
+const { getTenantId, isTenantIdSet } = require("@budibase/auth/tenancy")
+const { checkSlashesInUrl } = require("../utilities")
+const env = require("../environment")
+
+exports.syncUserInApps = async userId => {
+  if (env.isTest()) {
+    return
+  }
+  const request = { headers: {} }
+  request.headers[Headers.API_KEY] = env.INTERNAL_API_KEY
+  if (isTenantIdSet()) {
+    request.headers[Headers.TENANT_ID] = getTenantId()
+  }
+  request.headers["Content-Type"] = "application/json"
+  request.body = JSON.stringify({})
+  request.method = "POST"
+  const response = await fetch(
+    checkSlashesInUrl(env.APPS_URL + `/api/users/sync/${userId}`),
+    request
+  )
+  if (response.status !== 200) {
+    throw "Unable to sync user."
```

Comment:
I see a lot of `throw "string"` vs `throw new Error("string")`. Do we have a preference? 
from what I've seen the latter is more preferable as it results in a well formatted json message in response bodies, not sure if that's applicable here in this client call. I have sonar lint running in VS Code and it warns me about not wrapping these with `Error`

Reply:
Yeah, I usually just use the `throw "string"` because its just easier to use and expecting all errors to be wrapped is an error in an error waiting to happen (theres always going to be some libraries etc that don't wrap) - so my preference is to not wrap since the formatted version really isn't that useful.